src/components/TableCard/TableCard.jsx

```diff
@@ -16,6 +16,12 @@ import { generateTableSampleValues } from '../TimeSeriesCard/timeSeriesUtils';
 import { csvDownloadHandler } from '../../utils/componentUtilityFunctions';
 import CardToolbar from '../Card/CardToolbar';
 
+const ContentWrapper = styled.div`
```

Comment:
It looks like `ImageCard` also uses a very similar component. Could we abstract this out into one shared component for both instead of duplicating here?

Also we're discouraging use of styled-components and asking that new styles be put within a sass partial for reusability and ease of in-application overrides